### ISSUE TITLE
float-api-0.0.14 ModuleNotFoundError

### ISSUE THREAD

**Issue by lanjelot** [tobiasbp/float_api#30]
There is no `float_api.py` module in float-api-0.0.14
```
$ pip install float-api
Defaulting to user installation because normal site-packages is not writeable
Collecting float-api
  Obtaining dependency information for float-api from https://files.pythonhosted.org/packages/f5/e6/c36bb91a3051f1f8ac5b3a8e0ce0eefb431d3d251a5bff95e5e4ccbe8b9e/float_api-0.0.14-py3-none-any.whl.metadata
  Using cached float_api-0.0.14-py3-none-any.whl.metadata (3.0 kB)
Requirement already satisfied: requests in /usr/lib/python3.12/site-packages (from float-api) (2.28.2)
Requirement already satisfied: charset-normalizer<4,>=2 in /usr/lib/python3.12/site-packages (from requests->float-api) (3.2.0)
Requirement already satisfied: idna<4,>=2.5 in /usr/lib/python3.12/site-packages (from requests->float-api) (3.4)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/lib/python3.12/site-packages (from requests->float-api) (1.26.18)
Using cached float_api-0.0.14-py3-none-any.whl (3.0 kB)
Installing collected packages: float-api
Successfully installed float-api-0.0.14
$ find ~/.local/lib/python3.12/site-packages/ | grep -i float
./float_api-0.0.14.dist-info
./float_api-0.0.14.dist-info/INSTALLER
./float_api-0.0.14.dist-info/top_level.txt
./float_api-0.0.14.dist-info/METADATA
./float_api-0.0.14.dist-info/LICENSE
./float_api-0.0.14.dist-info/RECORD
./float_api-0.0.14.dist-info/REQUESTED
./float_api-0.0.14.dist-info/WHEEL
$
```

And to confirm, I downloaded the source distribution [float_api-0.0.14.tar.gz](https://files.pythonhosted.org/packages/1c/2d/591d430ab6c3dd370b137f01ae1dd7ead68ee56cfa75ae4fdc659d819b6d/float_api-0.0.14.tar.gz) that is listed on https://pypi.org/project/float-api/#files
```
$ tar tf float_api-0.0.14.tar.gz
float_api-0.0.14/
float_api-0.0.14/LICENSE
float_api-0.0.14/PKG-INFO
float_api-0.0.14/README.md
float_api-0.0.14/float_api/
float_api-0.0.14/float_api/float_api.egg-info/
float_api-0.0.14/float_api/float_api.egg-info/PKG-INFO
float_api-0.0.14/float_api/float_api.egg-info/SOURCES.txt
float_api-0.0.14/float_api/float_api.egg-info/dependency_links.txt
float_api-0.0.14/float_api/float_api.egg-info/requires.txt
float_api-0.0.14/float_api/float_api.egg-info/top_level.txt
float_api-0.0.14/pyproject.toml
float_api-0.0.14/setup.cfg
float_api-0.0.14/setup.py
float_api-0.0.14/tests/
float_api-0.0.14/tests/test_float_api.py
```

Looks like maybe the files `float_api/{__init__.py,float_api.py}` were missing when this release was packaged?

**Comment by lanjelot** [comment]
```
$ docker run -it python:3.9 bash
root@ef4e14a971c0:/# pip install float-api
[...]
root@ef4e14a971c0:/# find | grep -i float_api
./usr/local/lib/python3.9/site-packages/float_api-0.0.14.dist-info
./usr/local/lib/python3.9/site-packages/float_api-0.0.14.dist-info/INSTALLER
./usr/local/lib/python3.9/site-packages/float_api-0.0.14.dist-info/top_level.txt
./usr/local/lib/python3.9/site-packages/float_api-0.0.14.dist-info/METADATA
./usr/local/lib/python3.9/site-packages/float_api-0.0.14.dist-info/LICENSE
./usr/local/lib/python3.9/site-packages/float_api-0.0.14.dist-info/RECORD
./usr/local/lib/python3.9/site-packages/float_api-0.0.14.dist-info/REQUESTED
./usr/local/lib/python3.9/site-packages/float_api-0.0.14.dist-info/WHEEL
root@ef4e14a971c0:/# python
Python 3.9.18 (main, Dec 19 2023, 20:58:27) 
[GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from float_api import FloatAPI
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'float_api'
```

**Comment by lanjelot** [comment]
@tobiasbp you might wanna publish a new release :) and revoke 0.0.14 if that's even possible

**Comment by tobiasbp** [comment]
Hmmm... 0.0.14 was the first version released with GitHub actions... I'll have to figure out why files are missing. I'll see if I can remove broken release. Sorry.

**Comment by tobiasbp** [comment]
0.0.14 _yanked_ from _PyPi_. Deleted release and tag from this repo.

**Comment by tobiasbp** [comment]
I updated _setup.cfg_ and released 0.0.15. This time around, I actually tried out the new release :sweat_smile:
